firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/core/CrashlyticsControllerTest.java

```diff
@@ -83,7 +83,9 @@
       new FileFilter() {
         @Override
         public boolean accept(File pathname) {
-          return pathname.isDirectory() && !pathname.getName().equals("log-files");
+          return pathname.isDirectory()
+              && !pathname.getName().equals("log-files")
+              && !pathname.getName().equals("report-persistence");
```

Comment:
since we reference this string elsewhere, should we make it a constant?

Reply:
It's a test class, so not terribly worried about this.